### DISCUSSION TITLE
Cyberduck does only start on an RDSH (MS Terminal Server) when started as Administrator

### DISCUSSION THREAD

**Discussion by SvenRuben** [iterate-ch/cyberduck#15207] [Q&A]
Hello, I've installed the actual Version of Cyberduck on an Remote Desktop Session Host (Windows Terminal Server). It only starts when run "as Administrator" ... as a regular user it says: "The application Cyberduck has recently crashed"

Any suggestions, please?

Will that work with a license key?

Regards and thanks a lot in advance!

Sven

P.S. Update:

I did purchase a license, it's still the same, only starts when "run as administrator".

Please help!

Thanks, Sven.

**Comment by tatze96** [comment]
Please add the latest [crash report](https://docs.cyberduck.io/cyberduck/support/#crash-reports) for further investigation.

**Sidenote**: Ensure to attach the crash report file through the GitHub web interface, as it wouldn't be visible to us otherwise.
